### ISSUE TITLE
after firmware upgrade to 4.58.11.8391, camera feed doesn't work across internet anymore

### ISSUE THREAD

**Issue by guocity** [mrlt8/docker-wyze-bridge#1113]
after firmware upgrade to 4.58.11.8391, camera feed doesn't work across internet anymore

**Comment by jshg46** [comment]
I have a v3 doing the same exact thing. I can view feed in Wyze app but not Wyze docker.

**Comment by JA16122000** [comment]
Docker can't run the wyze bridge anymore overhere

**Comment by jshg46** [comment]
It would appear that some apps are having a relay problem after the newest firmware update. Which will not let the camera work if out of network.

**Comment by guocity** [comment]
yea, that sucks

> It would appear that some apps are having a relay problem after the newest firmware update. Which will not let the camera work if out of network.

yea, that sucks

**Comment by braynerz** [comment]
I am experiencing the same exact thing, updated two V3's and now cannot get their feed at all on docker, valid to mention that the cameras are not in the same network, I tried all the suggestions in this repository but no luck so far, hopefully wyze stops to push updates that break this awesome bridge

**Comment by jshg46** [comment]
It must not be hard to fix. I see Tinycam has already fixed it.

**Comment by jshg46** [comment]
Tony cam pro 17.2.1 updated via AppStore, fixed for me. I can now see cameras outside my home network.

**Comment by CowboyPilot** [comment]
It appears they are now charging to access "Web Live", any chance that is what has broken the integration?

**Comment by stevespaw** [comment]
Ouch I finally got this working again with the Wyze API key, And then stupidly I did a firmware update on the cameras and now the streams are broken.  I hope that this can be fixed. Can I roll back firmware with the app?

**Comment by scoob8000** [comment]
Same here.   Got tired of the app nagging me to update, now my two v2.s with 4.9.9.3006 cannot be reached.

**Comment by thereal-BLKMGK** [comment]
Okay, it's not just me :( I have a PTZ model that hasn't updated that still works but all of my remote V3 that I stupidly updated are now locked out. I'd sure love to get this fixed else I'm going to be forced to figure out a VPN to put these cameras on my "home" network. Everything in this place is working fine but I can no longer easily see what's going on at a remote site. I get alerts from Wyze but recording what's going on full time is way better! I hope this is something that can be solved...

**Comment by CowboyPilot** [comment]
Well I thought it may be the Cam Plus thing but my cameras are intermittently refreshing the image.  Stream doesn't work but I get a new image about every 12 hours from some of them

**Comment by jshg46** [comment]
Was really hoping to see an update for this issue by now.

**Comment by dddiggler** [comment]
> [#1099 (comment)](https://github.com/mrlt8/docker-wyze-bridge/issues/1099#issuecomment-1903957702)

Downgrading firmware would be an easy solution if the remote camera wasn't 1,200 miles across 5 states away!

Patience is a virtue, I'm sure we will get functionality back soon enough with Wyze-Bridge.  In the meantime, the official Wyze app will have to suffice!

**Comment by jshg46** [comment]
I have to agree, I have a few remote cameras that would be a pain to downgrade. Until this gets fixed I been using the webserver in Tinycam pro to view all my cams.

**Comment by thereal-BLKMGK** [comment]
Is a downgrade of the firmware even possible? My cameras are several hundred miles away currently but I'll be out there in a week or two. I didn't think a downgrade was possible but if it is I might do it to one or two of mine if this is still an issue. It's a shame Wyze won't let you use the live view that works on desktop on mobile :) Happy to hear another project figured out how to get around this, knowing it's possible is half the battle and is encouraging!

**Comment by CowboyPilot** [comment]
Yep use the SD card instructions.  I can confirm rolling back worked great for the V3, Pan v1 and Pan v2.  Just did it today and they are all working again.

**Comment by loups22** [comment]
Hi, do I need to leave the SD card in the camera when downgrading the firmware? does it copy the firmware to the camera or the SD card needs to be left inserted?

**Comment by CowboyPilot** [comment]
You put the file on the root of the SD card, unplug camera, put SD card in,
hold setup as you plug the cam in for about 5 seconds until the light turns
purple, takes about 2 minutes and it reboots.  You can then replace the SD
card with your old one.  Needs to be FAT formatted.  There are instructions
on the wyze site.  Once it's done go in and disable auto updates in
settings - device info - firmware

**Comment by wcarrx** [comment]
I was hopeful for a fix in 2.8.1 but after making necessary changes for the Wyze API change it works locally but not across the internet

**Comment by CowboyPilot** [comment]
Yeah I've just left my cams on the old firmware and it's still working.  I've got two servers one on latest and one on nightly.  My biggest issue is the one on latest stops serving the rtsp streams after a while.  I just set up a crown job to restart the docker every hour and it's been fine now

**Comment by dddiggler** [comment]
Bump - This is still an issue across the internet with wyze-bridge v.2.8.3

**Comment by mrlt8** [comment]
Unfortunately, this seems to be a known bug in the Wyze firmware. [B-44285](https://support.wyze.com/hc/en-us/articles/360032747431-Wyze-Software-and-Firmware-Bugs-and-Known-Issues): 

> B-44285: Unable to connect to live stream when not on local Wi-Fi


FYI, from my testing, I was able to connect to a remote camera over a VPN connection.

**Comment by wcarrx** [comment]
Yes

**Comment by dddiggler** [comment]
> Unfortunately, this seems to be a known bug in the Wyze firmware. [B-44285](https://support.wyze.com/hc/en-us/articles/360032747431-Wyze-Software-and-Firmware-Bugs-and-Known-Issues):
> 
> > B-44285: Unable to connect to live stream when not on local Wi-Fi
> 
> FYI, from my testing, I was able to connect to a remote camera over a VPN connection.

Thank you for all your fantastic work on this project.  Looks like the ball is in Wyze's court...

**Comment by CowboyPilot** [comment]
I still very much suspect this is tied into their policy that you have to have Wyze Cam Plus to view cameras through the webpage.

**Comment by f4llenz** [comment]
Adding to this, I paid for their Cam Plus Unlimited subscription so I can see the cameras on their website, it still didn't make the cameras work on wyze-bridge when they're not in the same network
